### ISSUE TITLE
Add `unsafe-to-chain-command` exception for `type` and `clear` commands when used together

### ISSUE THREAD

**Issue by fuxingloh** [cypress-io/eslint-plugin-cypress#132]
According to the [Cypress documentation on clear](https://docs.cypress.io/api/commands/clear#Command-Log), it is considered a correct usage to chain the `clear` command with the `type` command, and vice versa:

```ts
cy.get('input[name="name"]').clear().type('Jane Lane')
cy.get('input[name="name"]').type('Jane Lane').clear()
```

As of `v2.13.2`, the current lint rule does not allow this usage.

**Comment by nagash77** [comment]
@fuxingloh , Thank you for submitting your feature request.  Cypress Product Managers will review this request and consider it for future implementation.  Please remember we cannot guarantee this will be implemented in a specific timeframe or at all.  Please see the [Cypress App Product Roadmap](https://github.com/orgs/cypress-io/projects/13) for what we currently have in the works.

**Comment by websitevirtuoso** [comment]
yes. I just had to ingore this eslint rule. I have too many cases

      cy.getBySel('post.title').find('input').clear().type(newPost.title)

**Comment by dqiubread** [comment]
Any update on this change?

**Comment by corydeppen** [comment]
It's been a year since this was reported. Can a maintainer at least confirm that this is something that needs to be addressed in the plugin or if the docs aren't completely accurate and the plugin is reporting correctly?

**Comment by oracast** [comment]
it is dead. just move to playwright

**Comment by dqiubread** [comment]
> It's been a year since this was reported. Can a maintainer at least confirm that this is something that needs to be addressed in the plugin or if the docs aren't completely accurate and the plugin is reporting correctly?

This PR should fix it but it hasnt been merged. https://github.com/cypress-io/eslint-plugin-cypress/pull/141/files 
In the meantime my team decided to use patch-package with the changes in the PR to fix it.

**Comment by dqiubread** [comment]
> > It's been a year since this was reported. Can a maintainer at least confirm that this is something that needs to be addressed in the plugin or if the docs aren't completely accurate and the plugin is reporting correctly?
> 
> This PR should fix it but it hasnt been merged. https://github.com/cypress-io/eslint-plugin-cypress/pull/141/files In the meantime my team decided to use patch-package with the changes in the PR to fix it.

Sorry just realized this is on type and clear command chaining and not the chaining custom commands to other commands. My team decided to just un chain them.
